packages/core/src/balance/Balance.utils.ts

```diff
@@ -33,16 +58,99 @@ export function convertToTxUnit(balance: BN, power: number): BN {
   return new BN(balance).mul(new BN(10).pow(new BN(15 + power)))
 }
 
-export function asFemtoKilt(balance: BN): BN {
-  return convertToTxUnit(balance, 0)
+export const TRANSACTION_FEE = convertToTxUnit(new BN(125), -9)
+
+/**
+ * Safely converts the given [[BalanceNumber]] to a string, using the supplied methods,
+ * or it given a string checks for valid number representation.
+ *
+ * @param input [[BalanceNumber]] to convert.
+ * @returns String representation of the given [[BalanceNumber]].
+ * @throws On invalid number representation if given a string.
+ * @throws On malformed input.
+ */
+export function balanceNumberToString(input: BalanceNumber): string {
+  if (typeof input === 'string') {
+    if (!input.match(/^-?\d*\.?\d+$/)) {
```

Comment:
I guess this regex accepts also a number like `-.1`. Is this allowed? Or should also the expression before the dot be `\d+`?

Reply:
Yes you are correct, this accepts -.2 as -0.2. This behavior is insprired (copied) from the web3 ethjs.

Reply:
I haven't seen any test cases about that tho, so it might be good to add one for positive and one for negative at least 😊

Reply:
I implemented negative and positive tests for that, except for the case of it being actually -.2.

So thanks for bringing that up!